src/sagemaker/jumpstart/notebook_utils.py

```diff
@@ -246,6 +247,7 @@ def list_jumpstart_models(  # pylint: disable=redefined-builtin
     list_incomplete_models: bool = False,
     list_old_models: bool = False,
     list_versions: bool = False,
+    marketplace_model: bool = False,
```

Comment:
can we rename this: `list_marketplace_models`?

Reply:
also, i see it's set to `False` by default, but I see marketplace models get returned

Reply:
Updated this filtering logic to use filters, so we will do `list_jumpstart_models("model_type == marketplace")`